.github/workflows/ci.yml

```
@@ -38,9 +43,16 @@ jobs:
           cache-dependency-path: '**/pnpm-lock.yaml'
       - run: pnpm install --frozen-lockfile
       - run: pnpm run compile
-      - name: Install Playwright's dependencies
-        run: pnpm playwright install --with-deps
-      - run: pnpm run e2e
+      - name: Cache Playwright
+        uses: actions/cache@v3
+        id: playwright-cache
+        with:
+          path: ~/.cache/ms-playwright
+          key: ${{ runner.os }}-playwright-1.40.1
+      - name: Install Playwright
+        if: steps.playwright-cache.outputs.cache-hit != 'true'
+        run: pnpm dlx playwright@1.40.1 install --with-deps
```

Just curious what happens with `pnpm playwright install --with-deps`.
It causes some errors, right?

I regularly update dependencies, so I wonder if I need to modify this when I update playwright in package.json.

---

install --with-deps will install chrome, safari and firefox.

You will need to update this version too

---

```suggestion
        run: pnpm playwright install --with-deps
```
Let me try what will happen.